packages/react-percy-config/src/getWebpackSettings.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console */
+
+import chalk from 'chalk';
+import path from 'path';
+
+function sanitizeCustomWebpackConfig(config = {}) {
+  // Do not allow users to customize the entry config
+  if (config.entry) {
+    console.log(chalk.bold.yellow('Warning: ignoring `webpack.entry` field in percy.config.js.'));
```

Comment:
When reading the `webpack` config option from `percy.config.js`, we give users warnings about fields we're ignoring or about out-of-date syntax. But in these cases we automatically remove or rename the invalid settings for them so everything should work.